R/simulate_tools.R

```diff
@@ -290,7 +290,13 @@ spim_simulate_remove_dates_to <- function(obj, date) {
     return(obj)
   }
 
-  date <- date - 1
+  if (min(obj$date) == sircovid::sircovid_date(date)) {
```

Comment:
could you add a bit of docs here explaining the logic please?

Reply:
Done, thanks!

Reply:
I think we need a bit of further explanation about why it's specifically `date - 1`

Reply:
Perhaps things would be a lot clearer if the `date` produced by this if.... else...... had a name different from `date`?